components/Network/NetworkList.tsx

```diff
@@ -1,23 +1,25 @@
 import React, { FunctionComponent } from 'react'
-import { Network } from '@prisma/client'
+import { NetworkWithConnectionInfo } from 'services/networkService'
 import style from './network.module.css'
 
-interface IProps { networks: Network[] }
+interface IProps { networks: NetworkWithConnectionInfo[] }
 export default ({ networks }: IProps): FunctionComponent<IProps> => {
   return (
     <div className={style.network_ctn}>
       {networks.map(network => (
         <div key={network.id} className={style.card_wrapper}>
             <div className={style.network_card_text}>
               <div className={style.cardTitle}>{network.title}</div>
-              <div>Last block: X time ago</div>
-                {network.title === 'eCash' ? 
-                  <div className={style.cardStatus} style={{ color: '#04b504' }}>Connected</div> :
-                  <div className={style.cardStatus}>Disconnected</div>
+                {network.connected
+                  ? <>
+                    <div className={style.cardStatus} style={{ color: '#04b504' }}>Connected</div>
+                    <div>Last block: {network.minutesSinceLastBlock ?? '-'} {network.minutesSinceLastBlock === 1 ? 'minute' : 'minutes'} ago</div>
```

Comment:
Maybe we can make a new task to use a lib or custom function to display a nice bit of text here?

Something that covers:

 - 0 minutes: "just now"
 - >=60 minutes (switch to 1 hour / x hours)

Reply:
John and Scott have some experience with doing this in the past on other projects.